.changelog/16009.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:improvement
+Authentication: Warn when query token parameter is used for authentication.
```

Comment:
It's a little hard to discover but we have a set of "subsystems" we use in our changelogs. Adding something new won't break anything but it likely won't be grouped with similar PRs

```suggestion
acl: Warn when query token parameter is used for authentication.
```